### PR TITLE
Init monorepo

### PR DESCRIPTION
**Pull request by codewylle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"name": "@meshok/eslint-config",
```
Comment (finico) [inline package.json:2]: @codewylle man, you have 4-size tab here and 2-size space in other json files

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/node_modules/
```
Comment (finico) [inline .gitignore:1]: @codewylle only in the root, man?

File: package.json (hunk 1 above)
Comment (codewylle) [reply package.json:2]: Your meshok is strong, man!

File: .gitignore (hunk 2 above)
Comment (codewylle) [reply .gitignore:1]: I usually have .gitignore for each package

**Review by finico — APPROVED** [review]
You're really quick man!

**Pull request merged**